src/widgets/category-ingredient-container/category-ingredient-container.config.tsx

```diff
@@ -0,0 +1,30 @@
+//이후 각 카드 경로 설정 예정
+import { IngredientCategory } from '@/entities/ingredient/ingredient.type';
+
+export const INGREDIENT_CATEGORIES: readonly IngredientCategory[] = [
+  {
+    icon: '🌱',
+    title: '두부/콩류',
+
+    imageSrc: 'https://picsum.photos/id/237/350/200',
+    description: '두부, 렌틸콩, 병아리콩 등',
+  },
+  {
+    icon: '🌱',
+    title: '두부/콩류',
+    imageSrc: 'https://picsum.photos/id/237/350/200',
+    description: '두부, 렌틸콩, 병아리콩 등',
+  },
+  {
+    icon: '🌱',
+    title: '두부/콩류',
+    imageSrc: 'https://picsum.photos/id/237/350/200',
+    description: '두부, 렌틸콩, 병아리콩 등',
+  },
+  {
+    icon: '🌱',
+    title: '두부/콩류',
+    imageSrc: 'https://picsum.photos/id/237/350/200',
+    description: '두부, 렌틸콩, 병아리콩 등',
+  },
+] as const;
```

Comment:
as const 어서션은 읽기 전용으로 만들기 때문에 reaonly를 명시하지 않으셔도 괜찮을것 같습니다.
지금 사용하신 방법으로는 아래의 방식으로 다 뚤립니다.

```
INGREDIENT_CATEGORIES[0].title = "1234";
```

해당 내용에서 readonly는 의미가 없고 as const로는 타입 추론이 안되니 아래와 같이 수정하는게 어떨까요?
```suggestion
export const INGREDIENT_CATEGORIES= [
  {
    icon: '🌱',
    title: '두부/콩류',

    imageSrc: 'https://picsum.photos/id/237/350/200',
    description: '두부, 렌틸콩, 병아리콩 등',
  },
  {
    icon: '🌱',
    title: '두부/콩류',
    imageSrc: 'https://picsum.photos/id/237/350/200',
    description: '두부, 렌틸콩, 병아리콩 등',
  },
  {
    icon: '🌱',
    title: '두부/콩류',
    imageSrc: 'https://picsum.photos/id/237/350/200',
    description: '두부, 렌틸콩, 병아리콩 등',
  },
  {
    icon: '🌱',
    title: '두부/콩류',
    imageSrc: 'https://picsum.photos/id/237/350/200',
    description: '두부, 렌틸콩, 병아리콩 등',
  },
] satisfies IngredientCategory[];
```